src/components/scoresheets/scoresheets.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchScoresheet, fetchScoresheets, removeScoresheet } from '../../actions/scoresheet_actions';
+
+//=========================================
+// props / actions
+
+const mapStateToProps = (state) => {
+  return {
+    scoresheets: state.scoresheets,
+    currentUser: state.auth.currentUser,
+  }
+};
+
+const mapDispatchToProps = (dispatch) => ({
+  fetchScoresheet: (scoresheetId) => dispatch(fetchScoresheet(scoresheetId)),
+  fetchScoresheets: (userId) => dispatch(fetchScoresheets(userId)),
+  removeScoresheet: (scoresheetId) => dispatch(removeScoresheet(scoresheetId)),
+});
+
+//=========================================
+// component
+
+class Scoresheets extends React.Component {
+
+  static propTypes = {
+
+  }
+
+  static defaultProps = {
+    scoresheet: { name: "LOADING..." }
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Why are we putting things in the React state? In general, let's just use props and only user React state for something like a form where the user is making local changes.

Reply:
I've been advised (and have found) that keeping things in the React state guarantees updates upon changes - I'm thinking ahead to the idea that a user may be clicking on a "+" button and opens up a new scoresheet. In the render method, remind me - do we have access to props? Or would we need to define those in the constructor as a this.scoresheets to access them?

Reply:
React always rerenders on changes to props. The Redux-way to do things is to use Props EXCLUSIVELY, though we do tend to cheat that a little for forms.  We absolutely have access to props in the render method. Props should be your default most of the time!